README.md

```diff
@@ -183,10 +103,20 @@ import curry from 'crocks/helpers/curry'
 import isNumber from 'crocks/predicates/isNumber'
 import liftA2 from 'crocks/helpers/liftA2'
 import safe from 'crocks/Maybe/safe'
+```
+
+### Example
 
+Documentation references: [and][and-logic-crocks],
+[curry][curry-helpers-crocks],
+[predicates (isNumber)][predicate-docs],
+[liftA2][lifta2-helpers-crocks],
+[safe][safe-maybe-crocks].
+
+```javascript
 // divide :: Number -> Number
```

Comment:
This signature is wrong, but lets not block this PR for it

Reply:
I missed that, sorry.

The `safeDivide` signature is wrong too.